core/src/main/java/org/frankframework/management/bus/endpoints/TestPipeline.java

```diff
@@ -95,7 +95,7 @@ public BinaryMessage runTestPipeline(Message<?> message) {
 
 	// Does not support async requests because receiver requests are synchronous
 	private BinaryMessage processMessage(Adapter adapter, String payload, Map<String, String> threadContext, boolean expectsReply) {
-		String messageId = "testmessage" + UUIDUtil.createSimpleUUID();
+		String messageId = "testmessage-" + UUIDUtil.createSimpleUUID();
```

Comment:
En hier ook `MessageUtils.generateMessageId();` gebruiken.

Reply:
Ik wil het eigenlijk nog iets anders aanpakken, maar dan groeit het te ver buiten de scope van een PR voor ErrorMessageFormatter dus dat wil ik in een volgend PR doen.